HomeWork_03/DZ3.py

```diff
@@ -1,35 +1,32 @@
 #I love Python*42#
 
 # Zadanie 1
-x = ("I love Python")
+x = "I love Python"
 z = x*42
 print(z)
 
 #age_in_month#
 
 # Zadanie 2
-x = str("age_in_month")
-y = 37*12
-z = x+" "+str(y)
+x = month = 12
+y = age_in_year = 37
+z = age_in_month = x*y
 print(z)
 
 #Take age in "y" Zadanie 2#
 
 # Zadanie 3
-i = str(" age_in_years")
-j = str(i)+" " + str(y//12)
+i = " age_in_years"
```

Comment:
Age in years should be calculated as a result of the division of your age in months by 12, like this `age_in_years = age_in_month // 12` (divide without remainder)